Formula/dbus.rb

```diff
@@ -30,16 +36,16 @@ def install
     # Fix the TMPDIR to one D-Bus doesn't reject due to odd symbols
     ENV["TMPDIR"] = "/tmp"
 
-    system "./configure", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}",
-                          "--localstatedir=#{var}",
-                          "--sysconfdir=#{etc}",
-                          "--disable-xml-docs",
-                          "--disable-doxygen-docs",
-                          "--enable-launchd",
-                          "--with-launchd-agent-dir=#{prefix}",
-                          "--without-x",
-                          "--disable-tests"
+    system "./autogen.sh", "--disable-dependency-tracking",
```

Comment:
@MikeMcQuaid Just realized that I read your sentence with the question mark as part of the sentence and not the code bit. HEAD builds are the only ones that **require** `./autogen.sh`, but it seems to work for the stable and devel versions too.

@ilovezfs Yeah, that makes sense as to why only HEAD builds require it.